hack/tools/release/notes/generator.go

```diff
@@ -54,6 +55,7 @@ type notesEntry struct {
 	title    string
 	section  string
 	prNumber string
+	author   string
```

Comment:
I guess this depends where you make the filtering, but I think having it in `pr` is enough